website/notifications/utils.py

```diff
@@ -371,13 +373,68 @@ def get_parent_notification_type(node, event, user):
         return None
 
 
+def get_global_notification_type(global_subscription, user):
+    """
+    Given a global subscription (e.g. NotificationSubscription object with event_type
+    'global_file_updated'), find the user's notification type.
+    :param obj global_subscription: modular odm NotificationSubscription object
+    :param obj user: modular odm User object
+    :return: str notification type (e.g. 'email_transactional')
+    """
+    for notification_type in constants.NOTIFICATION_TYPES:
+        if user in getattr(global_subscription, notification_type):
+            return notification_type
+
+
+def check_if_all_global_subscriptions_are_none(user):
+    all_global_subscriptions_none = False
+    user_sunscriptions = get_all_user_subscriptions(user)
+    for user_subscription in user_sunscriptions:
+        if 'global' in user_subscription.event_name:
```

Comment:
Minor: It's theoretically possible for a non-global notification to have "global" in its name. To prevent possible clashing, it's safer to do `if user.subscription.startswith('global_')`.

This reminds me: it would be worth documenting the significance of "global_" in the field documentation (by convention, a code comment above the field).